src/Components/Modal/WebModal.tsx

```diff
@@ -0,0 +1,74 @@
+import * as React from "react";
+import * as ReactDOM from "react-dom";
+import { StyleSheet, TouchableOpacity, View } from "react-native";
+import { ModalProps } from "react-native-modal";
+import getShadowStyles from "src/Themes/getShadowStyle";
+import Colors from "src/Themes/Colors";
+
+class WebModal extends React.PureComponent<ModalProps> {
+  private documentBody = document.getElementsByTagName("body")[0];
+  private domElement = document.createElement("div");
+  static defaultProps = {
+    style: {
+      padding: "2%",
```

Comment:
`defaultProps` is not a place to provide styles to your component, because if any styles object will be provided to the component, all your component styles are overwritten.

Consider a following example:

```
class WebModal extends React.PureComponent<ModalProps> {
  static defaultProps = {
    style: {
      padding: "2%",
  }
}

<WebModal style={{backgroundColor: "black"}} />
```

In this scenario the padding property will disappear, even regardless that user didn't want to overwrite it.

`style` prop should be merged with the default styles of the component. So if any property is specified by user - it gets overwritten. If not - it should not be overwritten.